pkg/metrics/metrics.go

```diff
@@ -66,15 +71,15 @@ var pagingAlerts = []string{
 type Metrics interface {
 	UpdateMetricValidationFailed(string)
 	UpdateMetricValidationSucceeded(string)
-	UpdateMetricClusterCheckFailed(string)
-	UpdateMetricClusterCheckSucceeded(string)
+	UpdateMetricUpgradeoperatorHealthcheckSucceeded(string)
 	UpdateMetricScalingFailed(string)
 	UpdateMetricScalingSucceeded(string)
 	UpdateMetricUpgradeWindowNotBreached(string)
 	UpdateMetricUpgradeConfigSyncTimestamp(string, time.Time)
 	UpdateMetricUpgradeWindowBreached(string)
 	UpdateMetricUpgradeControlPlaneTimeout(string, string)
 	ResetMetricUpgradeControlPlaneTimeout(string, string)
+	UpdateMetricUpgradeoperatorHealthcheckFailed(string, string)
```

Comment:
```suggestion
	UpdateMetricHealthCheckFailed(string, string)
```
Suggesting similar naming as per equivalent `UpdateMetricHealthCheckSucceeded` metric suggestion. We can refactor the mocks and references accordingly